ci/provisioning/post_provision_config_nodes_LEAP_15.sh

```diff
@@ -79,16 +82,14 @@ post_provision_config_nodes() {
         done
     fi
     if [ -n "$INST_RPMS" ]; then
-        # shellcheck disable=SC2086
-        dnf -y erase $INST_RPMS
+        dnf -y erase "${INST_RPMS[@]}"
     fi
     rm -f /etc/profile.d/openmpi.sh
     rm -f /tmp/daos_control.log
-    dnf -y install $LSB_RELEASE
+    dnf -y install lsb-release
```

Comment:
Why are you undoing this installation from `$LSB_RELEASE`?

Reply:
fixed